scripts/run_script_1_extract_raw_data.R

```diff
@@ -21,7 +21,7 @@ main <- function() {
         tryCatch(
             process_tracker_file(paths, tracker_file, tracker_name, synonyms),
             error = function(e) {
-                logError("Could not process ", tracker_name, ". Error = ", e$message, ".")
+                logError(log_to_json("Could not process {values['tracker_name']}. Error = {values['e']}.", values = list(tracker_name = tracker_name, e = e$message), file = "scripts/run_script_1_extract_raw_data.R", line = 24, errorCode = "XDTWDGA", functionName = "main"))
```

Comment:
I think your approach is fine in general I just don't like that we have two functions that we need to call each time, so I guess it would be nicer to somehow have one logging function that does this internally...instead of `logInfo(log_to_json(...))` -> `log_info(...)`. It is also inconvenient that you first have to write the message and later pass the values as a list where you have to name it again, this is like double amount of work...I mean why do you want to use `values` as a list just to pass the names again? If I understand `glue` correctly, we can pass any variable as temporary variabel via keyword parameter, so your example would become:
```r
logError(
  log_to_json(
    message="Could not process {tracker_name}. Error = {errorMessage}.",
    tracker_name = tracker_name,
    errorMessage = e$message,
    file = "scripts/run_script_1_extract_raw_data.R", 
    line = 24, 
    errorCode = "XDTWDGA", 
    functionName = "main"
  )
)
```
`log_to_json` would have to accept any keyword parameter for this... If I understand this here (https://www.r-bloggers.com/2013/01/the-three-dots-construct-in-r/) correctly, you can do so by using `...` in R after all keyword parameter. So we can keep things like fileName etc., just the values for the message will be passed directly to glue via `...`. Does this make sense?